### PR TITLE
Fix CI issue

### PR DESCRIPTION
**Pull request by iAmWillShepherd**
I noticed the last commit ba9eca1 resulted in a broken build; this is because `MethodChannel.setMockMethodCallHandler` was replaced by `TestDefaultBinaryMessenger.setMockMethodCallHandler` ([ref](https://docs.flutter.dev/release/breaking-changes/mock-platform-channels#description-of-change)). The migration guide indicated that a specific package needed to be imported so the test could behave as expected.

---
This change is [Reviewable](https://reviewable.io/reviews/onesignal/onesignal-flutter-sdk/524)

### CODE REVIEW

**Review by jkasten2 — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/onesignal/onesignal-flutter-sdk/524#-:-MtFMQkP3ubpC1D_em4z:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @iAmWillShepherd)

**Comment by iAmWillShepherd** [comment]
@jkasten2 

Apologies, but I don't know the process for merging reviewed PRs. Who's responsibility is it to hit the button? I'm happy to do it, but don't want to do stuff out outside the process.

**Comment by jkasten2** [comment]
@iAmWillShepherd Sorry missed this, the author can merge after a maintainer approves the PR.
I am merging it in now however so it gets into the next release we shipping soon.

**Pull request merged**
